README.md

```diff
@@ -106,7 +106,13 @@ You can demo your changes by running the storybook demo locally with the followi
 yarn install
 yarn build
 cd packages/superset-ui-plugins-demo
-yarn storybook
+yarn storybook:run
+```
+
+Alternatively, you can demo your changes by using the following command while in `packages/superset-ui-plugins-demo`:
+
+```sh
+npm run storybook
```

Comment:
one final nit, this should be `yarn storybook`

Reply:
@etr2460 fixed